pom.xml

```diff
@@ -14,6 +14,20 @@
                <outputDirectory>C:\Program Files\payara-4.1.2.181\payara41\glassfish\domains\domain1\autodeploy</outputDirectory>
              </configuration>
         </plugin>
+       
+		   	<plugin>
+			<groupId>org.apache.maven.plugins</groupId>
+			<artifactId>maven-javadoc-plugin</artifactId>
+			<executions>
+				<execution>
+					<id>attach-javadocs</id>
+					<goals>
+						<goal>jar</goal>
```

Comment:
I think the "jar" goal is not a good option here, since we are not generating any jar file. A good option would be something like "compile" or "package"

Reply:
Did you generate the javadoc ?

Reply:
![Capture](https://user-images.githubusercontent.com/59320898/108179961-803a9f00-7106-11eb-811c-71beed0d6156.PNG)

yes it appears like this